app/src/main/java/com/lucas/medicaltools/MainView.kt

```diff
@@ -0,0 +1,15 @@
+package com.lucas.medicaltools
+
+import android.content.Intent
+import com.hannesdorfmann.mosby3.mvp.MvpView
+import com.jakewharton.rxrelay2.Relay
+import com.lucas.medical_equip.repository.MedicalTool
+import io.reactivex.Observable
+import io.reactivex.subjects.PublishSubject
+
+interface MainView: MvpView {
+    val onScreenLoadIntent: Relay<Intent>
```

Comment:
Delete this one: Only 1 required to represent your initial fetch. You can do it onCreate or onResume, choose the mechanism. onResume = fetch new data each time the screen appears again (e.g. Lock your phone and unlock it)